pkg/core/function.go

```diff
@@ -29,7 +29,7 @@ import (
 
 	"github.com/BurntSushi/toml"
 
-	"github.com/boz/go-logutil"
+	logutil "github.com/boz/go-logutil"
```

Comment:
why do we need this alias?

Reply:
Without it Goland automatically removes the import under me...